hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTable.scala

```diff
@@ -194,9 +194,14 @@ class TestAlterTable extends HoodieSparkSqlTestBase {
         val oldLocation = spark.sessionState.catalog.getTableMetadata(new TableIdentifier(tableName)).properties.get("path")
         spark.sql(s"alter table $tableName rename to $newTableName")
         val newLocation = spark.sessionState.catalog.getTableMetadata(new TableIdentifier(newTableName)).properties.get("path")
-        assertResult(false)(
-          newLocation.equals(oldLocation)
-        )
+        if (HoodieSparkUtils.isSpark3_2) {
```

Comment:
only use hoodieCatalog will set path to tblp

Reply:
@KnightChess spark 3.3 is supported. this also applies to 3.3 right?